h2o-py/h2o/h2o.py

```diff
@@ -294,6 +294,8 @@ def get_mem_size(mmint, mmgb):
         if not start_h2o: raise
         if ip and not (ip == "localhost" or ip == "127.0.0.1"):
             raise H2OConnectionError('Can only start H2O launcher if IP address is localhost.')
+        if https:
+            raise H2OConnectionError('Unable to start local server with https enabled. Consider disabling https.')
```

Comment:
If someone needs HTTPS they can start H2O jar manually. I would provide a link to the documentation that shows how to do that.

Reply:
Roger.

Reply:
Done.